org-roam.el

```diff
@@ -238,6 +238,14 @@ in temp buffers.  In cases where this occurs, we do know the file path, and pass
 it as FILE-PATH."
   (let ((file-path (or file-path
                        (file-truename (buffer-file-name)))))
+    (mapcan (lambda (linkvec)           ; Split any multi-cite citations
+              (if (string= (aref linkvec 2) "cite")
+                  (mapcar (lambda (ref) (vector (aref linkvec 0)
+                                                ref
+                                                (aref linkvec 2)
+                                                (aref linkvec 3)))
+                          (split-string (aref linkvec 1) ","))
```

Comment:
`org-ref` provides a utility: `org-ref-split-and-strip-string`

```
(defun org-ref-split-and-strip-string (string)
  "Split key-string and strip keys in STRING.
Assumes the key-string is comma delimited."
  (mapcar 'org-ref-strip-string (split-string string ",")))
```

which seems to do more robust replacements. Might be good to use this function instead.

Reply:
Uhm, I'm not sure if it's behaving as it should, though.
```el
(org-ref-split-and-strip-string "cite:ahrens2017,landy2012")
;; Result: ("cite:ahrens2017" "landy2012")
```

Reply:
Ah wait, maybe the `cite:` part is stripped before.

Reply:
Sounds good, I'll swap it to use that function

> Ah wait, maybe the `cite:` part is stripped before.

Yep this is stripped out earlier